utils/_legacy_check_future_compatible_signatures.py

```diff
@@ -173,17 +172,14 @@ def generate_hf_api_module() -> str:
     # Generate code with stubs
     generated_code = STUBS_SECTION_TEMPLATE_REGEX.sub(STUBS_SECTION_TEMPLATE.format(stubs=all_stubs_source), raw_code)
 
-    # Format (black+ruff)
+    # Format (ruff)
     return format_generated_code(generated_code)
 
 
 def format_generated_code(code: str) -> str:
     """
-    Format some code with black+ruff. Cannot be done "on the fly" so we first save the code in a temporary file.
+    Format some code with ruff. Cannot be done "on the fly" so we first save the code in a temporary file.
     """
-    # Format with black
-    code = black.format_file_contents(code, fast=False, mode=black.FileMode(line_length=119))
-
     # Format with ruff
     with tempfile.TemporaryDirectory() as tmpdir:
         filepath = Path(tmpdir) / "__init__.py"
```

Comment:
The call to `ruff` without arguments will not format but check. 
See #27144 where we heavily rely on the formatting of the `# Copied from`

Reply:
Good catch!
Fixed this in https://github.com/huggingface/huggingface_hub/pull/1824.